cls/SourceControl/Git/Extension.cls

```diff
@@ -49,15 +50,20 @@ Method UserAction(Type As %Integer, Name As %String, InternalName As %String, Se
         set Name = "%SourceMenu,Status"
     } 
 
+    if Type = 1, Name = 1 {
+        do ..AddToSourceControl(InternalName)
```

Comment:
Initial thought: We only want to do this automatically in basic mode
On further review: Actually we should just always do this automatically, you're right. I think you might also need to pick up Name = 7 though.